showcase/app/templates/components/app-header.hbs

```diff
@@ -0,0 +1,175 @@
+            <dd.Interactive @text="Documentation" @href="#" />
+            <dd.Interactive @text="Tutorials" @href="#" />
+            <dd.Interactive @text="Terraform Provider" @href="#" />
+            <dd.Interactive @text="Changelog" @href="#" />
+            <dd.Separator />
+            <dd.Interactive @text="Create support ticket" @href="#" />
+            <dd.Interactive @text="Give feedback" @href="#" />
+          </Hds::Dropdown>
+
+          <Hds::Dropdown class="hds-side-nav__dropdown" @listPosition="bottom-right" as |dd|>
+            <dd.ToggleIcon @icon="user" @text="user menu" />
+            <dd.Title @text="Signed In" />
+            <dd.Description @text="email@domain.com" />
+            <dd.Interactive @href="#" @text="Account Settings" />
+          </Hds::Dropdown>
+        </:utility>
+      </Hds::AppHeader>
+    </SG.Item>
+
+    <SG.Item @label="With max recommended phase 1 content">
+      <Hds::AppHeader>
+        <:logo>
+          <Hds::AppHeader::HomeLink @icon="hashicorp" @ariaLabel="HashiCorp home menu" @href="#" />
+        </:logo>
+
+        <:globalAfter>
+          <Hds::Dropdown class="hds-side-nav__dropdown" as |dd|>
+            <dd.ToggleButton @text="Choose an organization" @icon="org" />
+            <dd.Checkmark>
+              organizationName
+            </dd.Checkmark>
+          </Hds::Dropdown>
+        </:globalAfter>
+
+        <:utility>
+          <Hds::Button class="hds-side-nav__icon-button" @icon="search" @isIconOnly={{true}} @text="Search" />
+
+          <Hds::Dropdown class="hds-side-nav__dropdown" @listPosition="bottom-right" as |dd|>
+            <dd.ToggleIcon @icon="help" @text="help menu" />
+            <dd.Title @text="Help & Support" />
+            <dd.Interactive @text="Documentation" @href="#" />
+            <dd.Interactive @text="Tutorials" @href="#" />
+            <dd.Interactive @text="Terraform Provider" @href="#" />
+            <dd.Interactive @text="Changelog" @href="#" />
+            <dd.Separator />
+            <dd.Interactive @text="Create support ticket" @href="#" />
+            <dd.Interactive @text="Give feedback" @href="#" />
+          </Hds::Dropdown>
+
+          <Hds::Dropdown class="hds-side-nav__dropdown" @listPosition="bottom-right" as |dd|>
+            <dd.ToggleIcon @icon="user" @text="user menu" />
+            <dd.Title @text="Signed In" />
+            <dd.Description @text="email@domain.com" />
+            <dd.Interactive @href="#" @text="Account Settings" />
+          </Hds::Dropdown>
+        </:utility>
+      </Hds::AppHeader>
+    </SG.Item>
+
+    <SG.Item @label="With max recommended future content">
+      <Hds::AppHeader>
+        <:globalBefore>
+          {{! TODO: Rethink / Refactor buttons (Hds::SideNav::Header::IconButton) for use in AppHeader (& SideNav ) }}
+          <Hds::Button class="hds-side-nav__icon-button" @icon="grid-alt" @isIconOnly={{true}} @text="Product menu" />
```

Comment:
Note: Reusing class name from side-nav currently to start but we need to rethink what approach is best for both the "dark theme" and icon sizing.